native-code/development/index.md

```diff
@@ -59,7 +59,9 @@ For desktop development:
 
 See [Android][1] and [iOS][2] pages for separate instructions.
 
-
+NOTICE: if you get `Remote: Daily bandwidth rate limit exceeded for <ip>`, make
```

Comment:
NOTICE was previously highlighted, did you accidentally remove that?

Reply:
Done